docs/get-started/README.md

```diff
@@ -9,7 +9,7 @@ that aim to make working with external overlays, such as
 
 # Installation
 
-`ReactMapGL` requires `node >= v4` and `react >= 15.4`.
+`ReactMapGL` requires `node >= v6.4.0` and `react >= 15.4`.
```

Comment:
My understanding is node 6 is only required for dev? Is it still true after build? (v1 worked with 0.10)